src/main/java/core/basesyntax/GenerateArray.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class GenerateArray {
+    private static Colors[] color = Colors.values();
```

Comment:
better move this field to the colorRand() method

Reply:
done